### PR TITLE
(foreman.cp) add public dns to guest wifi

### PR DESCRIPTION
**Pull request by csilva-cl**
add public dns to guest wifi at the summit.

### CODE REVIEW

File: spec/hosts/nodes/foreman.cp.lsst.org_spec.rb
Comment (dtapiacl) [inline spec/hosts/nodes/foreman.cp.lsst.org_spec.rb:1]: I think the catalog is expecting the following order of, nameservers: ['1.0.0.1', '1.1.1.1', '8.8.8.8'] . As the rake is failing, other than this it looks good to me.

Comment (csilva-cl) [reply spec/hosts/nodes/foreman.cp.lsst.org_spec.rb:1]: the spec is expecting the order of the nameservers in the config ['1.1.1.1', '1.0.0.1', '8.8.8.8'], I fixed it in the last push

Comment (dtapiacl) [reply spec/hosts/nodes/foreman.cp.lsst.org_spec.rb:1]: I see, it looks good.

**Review by dtapiacl — APPROVED** [review]
LGTM.

**Pull request closed without merging**